ras_backstage/controllers/secure_messaging_controller.py

```diff
@@ -0,0 +1,99 @@
+import json
+import logging
+
+import requests
+from structlog import wrap_logger
+
+from ras_backstage import app
+from ras_backstage.exception.exceptions import ApiError
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def get_messages_list(encoded_jwt, label="", limit=1000):
+    logger.debug('Attempting to retrieve the messages list', label=label)
+    url = '{}{}'.format(app.config['RAS_SECURE_MESSAGING_SERVICE'], 'messages')
+    headers = {"Authorization": encoded_jwt}
+    response = requests.get(url, headers=headers, params={"label": label, "limit": limit})
+
+    if response.status_code != 200:
+        logger.error('Error retrieving the messages list', label=label, status_code=response.status_code)
+        raise ApiError(url, response.status_code)
+
+    logger.debug('Successfully retrieved the messages list', label=label)
+    return json.loads(response.text)
+
+
+def get_message(encoded_jwt, message_id, is_draft):
+    logger.debug('Attempting to retrieve message', message_id=message_id, is_draft=is_draft)
+    endpoint = 'draft/' if is_draft == 'true' else 'message/'
+    url = '{}{}{}'.format(app.config['RAS_SECURE_MESSAGING_SERVICE'], endpoint, message_id)
+    headers = {"Authorization": encoded_jwt}
+    response = requests.get(url, headers=headers)
+
+    if response.status_code != 200:
+        logger.error('Error retrieving the messages', status_code=response.status_code, message_id=message_id, is_draft=is_draft)
+        raise ApiError(url, response.status_code)
+
+    logger.debug('Successfully retrieved message', message_id=message_id, is_draft=is_draft)
```

Comment:
Why is this level debug logging but all others are info?

Reply:
The way I've broke up the logging is that on the top level in Resources everything is logged at info level so we see requests coming in/out. All these controller functions are at debug level for more detail on what requests are being made

Reply:
That's fair enough. Why are the functions below done in info?